packages/studio-plugin/src/utils/ComponentTreeHelpers.ts

```diff
@@ -186,4 +186,44 @@ export default class ComponentTreeHelpers {
 
     return descendants;
   }
+
+  /**
+   * Returns the component UUID that is closest common parent of the two target components.
+   * This is similar to the lowest common ancestor except it cannot be one of the targets.
+   */
+  static getLowestParentUUID(
+    targetOneUUID: string | undefined,
+    targetTwoUUID: string | undefined,
+    componentTree: ComponentState[]
+  ) {
+    const LCAComponent = ComponentTreeHelpers.getLowestCommonAncestorComponent(
+      targetOneUUID,
+      targetTwoUUID,
+      componentTree
+    );
+    const TargetIsLCA =
+      targetOneUUID === LCAComponent?.uuid ||
+      targetTwoUUID === LCAComponent?.uuid;
+    return TargetIsLCA ? LCAComponent?.parentUUID : LCAComponent?.uuid;
```

Comment:
instead of doing this logic to get the parent if the LCA is one of the targets, can we just pass the parent of each target to getLowestCommonAncestorComponent to begin with?

Reply:
oh genius! yes changed